democracy/migrations/0030_migrate_translatable_fields.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.10 on 2016-12-08 12:00
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+from django.core.exceptions import ObjectDoesNotExist
+
+IMPORT_LANGUAGE_CODE = 'fi'  # Assume that the existing content is in Finnish
```

Comment:
Actually there are one or two Hearings in Swedish or English already present. Could we pick the Section, Hearing and SectionImage language using `detect_lang`? Labels and ContactPerson don't have long enough strings for the detection to be reliable.